src/Orleans/Runtime/GrainInterfaceMap.cs

```diff
@@ -76,8 +76,11 @@ public override string ToString()
         private readonly bool localTestMode;
         private readonly HashSet<string> loadedGrainAsemblies;
 
-        public GrainInterfaceMap(bool localTestMode)
+        private readonly PlacementStrategy defaultPlacementStrategy;
```

Comment:
I wonder would it be useful to make the default a public property. With the idea the default would be visible.